media-gfx/pencil/pencil-0.6.4.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop qmake-utils xdg-utils
+
+DESCRIPTION="2D animation and drawing program based on Qt5"
+HOMEPAGE="https://www.pencil2d.org/"
+SRC_URI="https://github.com/pencil2d/${PN}/archive/v${PV/_/-}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Could you verify the license wrt https://devmanual.gentoo.org/general-concepts/licenses/index.html#gpl-x-vs-gpl-x+ ?